scene/3d/light_3d.cpp

```diff
@@ -414,11 +416,8 @@ void DirectionalLight3D::_bind_methods() {
 DirectionalLight3D::DirectionalLight3D() :
 		Light3D(RenderingServer::LIGHT_DIRECTIONAL) {
 
-	set_param(PARAM_SHADOW_NORMAL_BIAS, 0.8);
```

Comment:
You removed the defaults here and then never replaced them

Reply:
ah..